topology/query.go

```diff
@@ -0,0 +1,34 @@
+package topology
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/flanksource/incident-commander/api"
+	"github.com/flanksource/incident-commander/db"
+)
+
+func QueryRenderComponents(ctx context.Context, systemTemplateID string) ([]api.Renderers, error) {
```

Comment:
@moshloop Should we keep this in `db/topology.go` as we keep all the database interactions in the db package

Reply:
I assumed that was the convention I am supposed to follow. I took `pkg/snapshot` as a reference but looks like `pkg/rules` also does it.